Makefile

```diff
@@ -369,21 +369,22 @@ ifneq ($(RUST),0)
 $(MSGGEN_GENALL)&: doc/schemas/*.request.json doc/schemas/*.schema.json
 	PYTHONPATH=contrib/msggen python3 contrib/msggen/msggen/__main__.py
 
+GRPC_PATH = contrib/pyln-grpc-proto/pyln/grpc
 GRPC_GEN = \
-	contrib/pyln-grpc-proto/pyln/grpc/node_pb2.py \
-	contrib/pyln-grpc-proto/pyln/grpc/node_pb2_grpc.py \
-	contrib/pyln-grpc-proto/pyln/grpc/primitives_pb2.py
+	$(GRPC_PATH)/node_pb2.py \
+	$(GRPC_PATH)/node_pb2_grpc.py \
+	$(GRPC_PATH)/primitives_pb2.py
 
 ALL_TEST_GEN += $(GRPC_GEN)
 
 $(GRPC_GEN)&: cln-grpc/proto/node.proto cln-grpc/proto/primitives.proto
-	python -m grpc_tools.protoc -I cln-grpc/proto cln-grpc/proto/node.proto --python_out=contrib/pyln-grpc-proto/pyln/grpc/ --grpc_python_out=contrib/pyln-grpc-proto/pyln/grpc/ --experimental_allow_proto3_optional
-	python -m grpc_tools.protoc -I cln-grpc/proto cln-grpc/proto/primitives.proto --python_out=contrib/pyln-grpc-proto/pyln/grpc/ --experimental_allow_proto3_optional
+	python -m grpc_tools.protoc -I cln-grpc/proto cln-grpc/proto/node.proto --python_out=$(GRPC_PATH)/ --grpc_python_out=$(GRPC_PATH)/ --experimental_allow_proto3_optional
+	python -m grpc_tools.protoc -I cln-grpc/proto cln-grpc/proto/primitives.proto --python_out=$(GRPC_PATH)/ --experimental_allow_proto3_optional
 	# The compiler assumes that the proto files are in the same
 	# directory structure as the generated files will be. Since we
 	# don't do that we need to path the files up.
-	find contrib/pyln-grpc-proto/pyln/ -type f -name "*.py" -print0 | xargs -0 sed -i'.bak' -e 's/^import \(.*\)_pb2 as .*__pb2/from pyln.grpc import \1_pb2 as \1__pb2/g'
-	rm -f contrib/pyln-grpc-proto/pyln/*.py.bak
+	find $(GRPC_PATH)/ -type f -name "*.py" -print0 | xargs -0 sed -i'.bak' -e 's/^import \(.*\)_pb2 as .*__pb2/from pyln.grpc import \1_pb2 as \1__pb2/g'
+	rm -f $(GRPC_PATH)/*.py.bak
```

Comment:
So this last line is the real fix, since the find kinda worked, but the rm didn't?

Reply:
Oh no, I missed that the `find` uses a different path (up one level)... and there's even a comment, oops.

seems like the real fix would be to have `rm` do a recursive hunt also? ok let me try again here. rip.